pkg/function/restore_csi_snapshot_test.go

```diff
@@ -95,7 +95,7 @@ func (testSuite *RestoreCSISnapshotTestSuite) TestRestoreCSISnapshot(c *C) {
 		originalPVC := getOriginalPVCManifest(testSuite.pvcName, testSuite.storageClass)
 		createPVC(c, testSuite.namespace, originalPVC, fakeCli)
 
-		err = fakeSnapshotter.Create(ctx, testSuite.snapName, testSuite.namespace, testSuite.pvcName, &testSuite.volumeSnapshotClass, false, nil, nil)
+		err = fakeSnapshotter.Create(ctx, testSuite.pvcName, &testSuite.volumeSnapshotClass, false, metav1.ObjectMeta{Name: testSuite.snapName, Namespace: testSuite.namespace})
```

Comment:
Here as well, let's create the snapshotMeta first and then pass that to this method call.